package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@wink-financial/wink-assets",
-  "version": "2.5.8",
+  "version": "2.6.0",
```

Comment:
you could use 2.5.9 if it's just an image chain, minor version is used when adding new networks or changing that affect functionality but are not breaking changes.

Reply:
Done!